package-templates/react/package.json

```diff
@@ -24,8 +24,8 @@
     "check": "npm run lint && npm run test",
     "lint": "eslint -f codeframe \"./**/*.{js,jsx}\"",
     "build": "rollup --config rollup.config.js",
-    "start": "start-storybook -p 3000 --ci",
-    "start:ssr-preview": "SSR=true start-storybook -p 3000 --ci --preview-url=/ssr-iframe",
+    "start": "start-storybook -p 3000 --ci --config-dir ../../.storybook",
+    "start:ssr-preview": "SSR=true start-storybook -p 3000 --ci --preview-url=/ssr-iframe --config-dir ../../.storybook",
```

Comment:
Idea: Extract to `../../scripts/start-storybook` and `../../scripts/stop-storybook`

Reply:
or start from root and serve all storybooks?